.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,57 @@
+name: Java CI
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: 'postgres'
+          POSTGRES_DB: test_db
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0 # This is wasteful, but needed for git describe
```

Comment:
@wetneb Wait, I'm confused here on this `fetch-depth` line.  I don't see the git describe need. Where at?  I see `git describe` used in the `refine` script for `mac_dist()` which says no longer used in a comment, and then also for `broker_build()` and `broker_appengine_build()`.  Are those even run or needed now since we now do "mvn package" ?  For our PR workflow, I think we just care about what the last tag was on master, no?  Or even just 'heads/master' ? 
```
E:\GitHubRepos\OpenRefine>git describe --tags --first-parent --all
heads/master
```
BTW, we also have access to the environment variable in the runner at any time with **GITHUB_REF** which represents the PR merge branch `refs/pull/:prNumber/merge` as shown in docs https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request

Reply:
Good point, I just took that from the other workflow, it's not needed here indeed.